portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,106 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#background-block {
+    background-color: #e897c3 ;
+}
+
+.logo {
+  position: absolute;
+  left: 15px;
+  top: 105px;
+}
+
+.headshot {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+.h1 { 
+    font-family: "Rockwell Extra Bold", "Rockwell Bold", monospace; 
```

Comment:
From https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks:

Use single ('') rather than double ("") quotation marks for attribute selectors and property values.